packages/modal/src/useModal.ts

```diff
@@ -5,7 +5,7 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import { useRef, useState } from 'react';
+import { useRef, useState, KeyboardEvent, MouseEvent } from 'react';
```

Comment:
So simply importing these fixes the typing issue? Im confused by the brevity here. 🤔

Reply:
Yeah, I can clarify further. Consumers may [pass](https://github.com/zendeskgarden/react-components/blob/6bcf661282e629b3482c214192475f47174c1aa7/packages/modals/demo/stories/ModalStory.tsx#L70) `onClose` to React elements so this needs to satisfy synthetic event constraints. Otherwise, consumers are forced to [cast](https://github.com/zendeskgarden/react-components/blob/6bcf661282e629b3482c214192475f47174c1aa7/packages/modals/demo/stories/ModalStory.tsx#L26) the handler event as `any` to side step synthetic constraints.

<img width="1119" alt="Screen Shot 2022-02-28 at 11 08 21 AM" src="https://user-images.githubusercontent.com/1811365/156061586-e5d3861a-1371-4871-af47-a7f64b38716c.png">

Reply:
> type needs to be imported from React (synthetic event) rather than using the global (vanilla JavaScript) event types.

Gotcha 👌